README.md

```diff
@@ -87,4 +87,6 @@ A dummy implementation of the API, containing sample XML content in multiple lan
 - `journal` does not convert all content to HTML
 - `dashboard` is missing some event name translations
 - `article-store` persists article versions before the workflows are run
+- `article-store` doesn't validate input
```

Comment:
Was by design. Do we have a strategy for accessing the schemas(s)? if so this can be implemented.

Reply:
It would be configured with a schema somehow (currently seems to take it as input). Wouldn't worry about it too much.